templates/paragraph--airtable--service-catalog.html.twig

```diff
@@ -0,0 +1,111 @@
+ *   - paragraph.getOwnerId(): The user ID of the paragraph author.
+ *   See Drupal\paragraphs\Entity\Paragraph for a full list of public properties
+ *   and methods for the paragraph object.
+ * - content: All paragraph items. Use {{ content }} to print them all,
+ *   or print a subset such as {{ content.field_example }}. Use
+ *   {{ content|without('field_example') }} to temporarily suppress the printing
+ *   of a given child element.
+ * - attributes: HTML attributes for the containing element.
+ *   The attributes.class element may contain one or more of the following
+ *   classes:
+ *   - paragraphs: The current template type (also known as a "theming hook").
+ *   - paragraphs--type-[type]: The current paragraphs type. For example, if the paragraph is an
+ *     "Image" it would result in "paragraphs--type--image". Note that the machine
+ *     name will often be in a short form of the human readable label.
+ *   - paragraphs--view-mode--[view_mode]: The View Mode of the paragraph; for example, a
+ *     preview would result in: "paragraphs--view-mode--preview", and
+ *     default: "paragraphs--view-mode--default".
+ * - view_mode: View mode; for example, "preview" or "full".
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
+ *
+ * @see template_preprocess_paragraph()
+ *
+ * @ingroup themeable
+ */
+#}
+{%
+  set classes = [
+    'paragraph',
+    'paragraph--type--' ~ paragraph.bundle|clean_class,
+    view_mode ? 'paragraph--view-mode--' ~ view_mode|clean_class,
+    not paragraph.isPublished() ? 'paragraph--unpublished',
+    'rh-service-catalog'
+  ]
+%}
+{%
+  set airtableConfig = {
+    "table": "tbltFlPWqzu0V8RuD",
+    "view": "viwCEu6klnwbAYCWE",
+    "gutter": 40,
+    "equalHeight": true,
+    "filters": {
+      "Core Service": {
+        "name": "Service"
+      },
+      "Clients": {
+        "name": "Audience"
+      }
+    },
+    "format": {
+      "Core Service": {
+        "type": "Array",
+        "options": {
+          "template_id": "metadata-item",
+          "separator": " | ",
+        }
+      },
+      "Clients": {
+        "type": "Array",
+        "options": {
+          "template_id": "metadata-item",
+          "separator": " | ",
+        }
+      }
+    }
+  }
+%}
+{% block paragraph %}
+  <div{{ attributes.addClass(classes) }}>
+    {% block content %}
+      {# Filters #}
+      <div id="airtable-list-filters"></div>
+      <div id="airtable-list" data-config='{{ airtableConfig | json_encode }}'>
+        <p id="airtable-list-loader">Loading service catalog...</p>
+        <p id="airtable-list-no-results" style="display: none;">There are no results matching your current criteria.</p>
+        <ul id="airtable-list-record-wrapper">
+        </ul>
+      </div>
```

Comment:
Is it possible to have more than one of these paragraphs on a page? 
Normally we encourage the use of CSS classes over Ids. Even if there is js targeting it but I get it if this is a requirement of another lib.

Reply:
Good observation. It wouldn't make sense to have multiple of them. The javascript isn't setup to support multiple at the moment anyway.